pkg/apis/authentication/types.go

```diff
@@ -162,3 +162,24 @@ type BoundObjectReference struct {
 	// UID of the referent.
 	UID types.UID
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// SelfSubjectAttributesReview contains the user information that the kube-apiserver has about
+// the user who originated the request.
+type SelfSubjectAttributesReview struct {
+	metav1.TypeMeta
+	// Standard list metadata.
```

Comment:
looks like a copy/paste typo... this is object metadata, not list metadata

Reply:
Fixed.